### DISCUSSION TITLE
Welcome to smarty-vscode-support Discussions!

### DISCUSSION THREAD

**Discussion by aswinkumar863** [aswinkumar863/smarty-vscode-support#173] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by asharris** [comment]
I use `{%` and `%}` for start and end of a smarty block so it does not interfere with JavaScript rather than having to use `{literal}…{/literal}`.
Is there a way I can tell smarty-vscode to support that? If so, how? it would make comment blocks so much easier to use.
Antony
